src/envoy/watcher.clj

```diff
@@ -16,49 +20,74 @@
   (-> (http/get path (core/with-ops ops))
       index-of)))
 
+(defn- data->channel
```

Comment:
`data` is not a good name for a variable
`put-if-open!` would be. better name for a function, since that is what it is doing

Reply:
:+1: @tolitius